.github/ISSUE_TEMPLATE/bug-core.md

```diff
@@ -1,13 +1,11 @@
 ---
+
 name: "\U0001F6A8 Bug: Core"
 about: A bug that occurs in Slate's core logic, on all platforms
 title: ''
 labels: bug
 assignees: ''
-
----
-
-**Description**
+---**Description**
```

Comment:
we did a rebase together off the `main` branch. looks like the `yarn fix` command broke this? @adisen and I will check again...